[task 3]Feature/filipe/judge

FiAZV opened the pull request.

FiAZV committed:
Added judge.py file

the file judge.py was added to the project to complete task 3

FiAZV committed:
Added judge.py file

the file judge.py was added to the project to complete task 3

FiAZV committed:
Added judge.py file

the file judge.py was added to the folder Filipe to complete task 3

FiAZV committed:
Resolvendo conflitos

resolvendo os conflitos que foram apresentados no último PR

FiAZV committed:
Enviando apenas os arquivos necessários

haviam arquivos na pasta Jose que foram importados atraves de um merge da developque criaram um conflito, o qual já foi resolvido, estou apagando essa pasta da branch feature/filipe/judge e enviando apenas os arquivos my_function.py e judge.py da pasta Filipe, alem do README.md

FiAZV committed:
Adicionando novamente os arquivos deletados

as pastas Jose e George tinham sido deletadas por mim por um erro meu, estou adicionando elas novamente

FiAZV committed:
Consertando o conflito em Filipe/my_function

GeorgeJuniorGG commented on Filipe/judge.py:
```diff
@@ -0,0 +1,29 @@
+# a ideia é que o programa conte cada letra de uma frase ou palavra
+
+sentenca = input ("Digite a sentença sem números assentos ou caracteres específicos: ")
```
Acho que faltou uma vírgula entre números e assentos

ThalesRogerio commented on Filipe/judge.py:
```diff
@@ -0,0 +1,29 @@
+    if sentenca[j] == " " or sentenca[j] in numeros:
+        continue
+    for i in range(len(letras)):
+        if sentenca[j] == letras[i]:
```
Me parece que esse codigo pode ser bem simplificado se você usar um dicionario ao inves de 2 vetores. Um dicionario to tipo { letra : contador}. 

Você pode criar um bem facilmente com 
`dictionary={character : 0 for character in letras}`

FiAZV committed:
Simplificando o código usando dicionário e corrigindo erros de português

implementei a sugestão de ao invés de usar dois vetores, usar um dicionário e corrigi a virgula no input.
